tests/unit/tickstore/test_toplevel.py

```diff
@@ -131,6 +132,8 @@ def test_slice_list_of_dicts(start, end, expected_start_index, expected_end_inde
     assert expected == result
 
 
+@pytest.mark.xfail(sys.version_info >= (3,),
+                   reason="python3 issue with mock 1.0.1")
```

Comment:
If we can upgrade mock most of them will work. Most the errors I was getting show they were resolved in much later versions than what we use. If we can do that, we can also support python 3.4